packages/ui/src/components/GalleryMedia/GalleryMedia.tsx

```diff
@@ -45,24 +43,22 @@ export function GalleryMedia({ className, gallery, locale }: Props) {
             >
                 <p className="title-x-small text-gray-100">{name}</p>
                 <div className="flex items-center justify-between">
-                    {Boolean(images_number || videos_number) && (
+                    {Boolean(imagesNumber || videosNumber) && (
                         <div className="flex items-center gap-3">
-                            {Boolean(images_number) && (
+                            {Boolean(imagesNumber) && (
                                 <span className="flex text-gray-100">
                                     <PhotoIcon className="w-5 h-5 mr-2" />
-                                    {/* TODO: Add translations */}
-                                    {videos_number === 0
-                                        ? `${images_number} Images`
-                                        : images_number}
+                                    {videosNumber === 0
+                                        ? `${imagesNumber} ${intl['images.title'] ?? 'Images'}`
+                                        : imagesNumber}
                                 </span>
                             )}
-                            {Boolean(videos_number) && (
+                            {Boolean(videosNumber) && (
                                 <span className="flex text-gray-100">
                                     <VideoCameraIcon className="w-5 h-5 mr-2" />
-                                    {/* TODO: Add translations */}
-                                    {images_number === 0
-                                        ? `${videos_number} Videos`
-                                        : videos_number}
+                                    {imagesNumber === 0
+                                        ? `${videosNumber} ${intl['videos.title'] ?? 'Videos'}`
```

Comment:
The assumption that `{x} videos` have the same words order in all languages is invalid. Think RTL languages, where it can be `xxxxx {x}`, or _maybe_ there is a language where this is written as `xxxxx {x} yyyyy`. The general thumb rule is that you should never combine sentences of parts manually and instead always interpolate i18n dictionary strings.

Please leave a `FIXME` comment here, we'll look into it later.